{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -334,6 +335,13 @@
 # django-cors-headers - https://github.com/adamchainz/django-cors-headers#setup
 CORS_URLS_REGEX = r"^/api/.*$"
 
+# By Default swagger ui is available only to admin user. You can change permission classs to change that
+SPECTACULAR_SETTINGS = {
+    "TITLE": "API documentation for{{ cookiecutter.project_slug }}",
+    "DESCRIPTION": "Lets rock and then roll!",
+    "VERSION": "1.0.0",
+    "SERVE_PERMISSIONS": ["rest_framework.permissions.IsAdminUser"],
+}
```

Comment:
Maybe it would be good to also add server. This will make the generated commands already point to the "correct" domain.


```suggestion
    "SERVERS": [
        {
            "url": "https://127.0.0.1:8000",
            "description": "Local Development server"
        },
#        {
#            "url": "https://staging.{{ cookiecutter.domain_name }}",
#            "description": "Development server"
#        },
        {
            "url": 'https://{{ cookiecutter.domain_name }}",
            "description": "Production server"
        }
    ],
}
```

Reply:
That's a good suggestion. We can include that as well.